smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/IndexGenerator.java

```diff
@@ -45,6 +47,16 @@ static void writeIndex(
         ProtocolGenerator protocolGenerator,
         TypeScriptWriter writer
     ) {
+        ServiceShape service = settings.getService(model);
+        Optional<String> docs = service.getTrait(DocumentationTrait.class)
+                .map(DocumentationTrait::getValue);
+        writer.write("/* eslint-disable */");
+        docs.ifPresent(documentationTrait -> writer.writeDocs(
```

Comment:
nit: since the service and its docs aren't used again, I might do this instead:
```
writer.write("/* eslint-disable */");
settings.getService(model).getTrait(DocumentationTrait.class).ifPresent(trait ->
    writer.writeDocs(trait.getValue() + "\n\n" + "@packageDocumentation"));
```
And then you can leave out the `Optional` import.